x/tokenregistry/keeper/migrations.go

```diff
@@ -24,3 +26,45 @@ func (m Migrator) MigrateToVer2(ctx sdk.Context) error {
 	m.keeper.SetRegistry(ctx, registry)
 	return nil
 }
+
+func (m Migrator) MigrateToVer4(ctx sdk.Context) {
```

Comment:
I'd like to know how do you confirm the version is 4? when and where the method will be called?

Reply:
This is based on the latest version on develop right now , This function is called based on the the consensus version in module.go